scripts/reviews/update_auth_groups.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Since groups are created [when new providers are added](https://github.com/chrisseto/osf.io/pull/26/files/700a2df93da565db99b62870b18f0242f27e124e#diff-03133408b78d570d4fe79642d38b06c8R106), it's redundant for new databases and seemed wrong to put in a migration. But I guess it is idempotent, so may as well...

This script is still handy, though, if we ever want to change a group's permissions. We can just edit `reviews.permissions.GROUPS` and run it.